src/parse.c

```diff
@@ -0,0 +1,78 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   parse.c                                            :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gwolf <gwolf@student.42vienna.com>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/11/12 16:02:54 by gwolf             #+#    #+#             */
+/*   Updated: 2023/11/20 13:03:50 by gwolf            ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "parse.h"
+
+t_err	ft_parse_file(char *filename, t_entities *ents, t_cam *cam)
```

Comment:
I'd call it scene instead of ents to keep consistency

Reply:
Changed